src/main/java/core/basesyntax/controllers/car/CreateCarController.java

```diff
@@ -0,0 +1,44 @@
+package core.basesyntax.controllers.car;
+
+import core.basesyntax.lib.Injector;
+import core.basesyntax.model.Car;
+import core.basesyntax.model.Manufacturer;
+import core.basesyntax.service.CarService;
+import core.basesyntax.service.ManufacturerService;
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class CreateCarController extends HttpServlet {
+    private static Injector injector = Injector.getInstance("core.basesyntax");
```

Comment:
```suggestion
    private static final Injector injector = Injector.getInstance("core.basesyntax");
```
check that it is static and final in all your controllers